swarm-production/docker-stack.yml

```diff
@@ -52,7 +50,8 @@ services:
       METADATA_HOST: s3-metadata
       REDIS_HOST: cache
       ENDPOINT: "${ENDPOINT:-zenko}"
-      MANAGEMENT_ENDPOINT: 'https://api.zenko.io'
+      ## Temp change to dev endpoint for Clueso testing
+      MANAGEMENT_ENDPOINT: 'https://api-dev.private.zenko.io'
```

Comment:
this doesn't seem to be in a dropme commit

Reply:
I ended up squashing all the commits indiscriminately - I was going to just change the line later and squash again xD